src/setIntersection.js

```diff
@@ -0,0 +1,15 @@
+export default function setInstersection(array, intersect){
+  let result = []
+  for(let i=0;i<array.length;i++){
+    let intersection = false
+    for(let j=0;j<intersect.length;j++){
+      if(array[i] === intersect[j]){
+        intersection = true
+      }
+    }
```

Comment:
if you have time later in the week I wonder if there's a way to refactor this to be faster than `On^2` (nested loops, having to go through each array twice)
Don't worry about it unless you have time, this is a good enough first pass.